ethereum/core/build.gradle

```diff
@@ -54,6 +58,7 @@ dependencies {
   implementation 'org.apache.tuweni:tuweni-concurrent'
   implementation 'org.apache.tuweni:tuweni-units'
   implementation 'org.apache.tuweni:tuweni-rlp'
+  implementation 'tech.pegasys:jc-kzg-4844:0.1.0'
```

Comment:
Versions should not be listed in child build files.  Please add an entry to `<root>/gradle/versions.gradle`